lib/recommended.yaml

```diff
@@ -42,9 +46,13 @@ linter:
     - type_init_formals
     - unnecessary_brace_in_string_interps
     - unnecessary_const
+    - unnecessary_constructor_name
     - unnecessary_getters_setters
+    - unnecessary_late
     - unnecessary_new
+    - unnecessary_null_aware_assignments
```

Comment:
This one is also missing an auto fix, https://github.com/dart-lang/lints/issues/66.

Reply:
Now fixed.